src/lib/chat/editOperations.ts

```
@@ -0,0 +1,215 @@
+    skipped,
+  };
+}
+
+/**
+ * Generates a human-readable summary of what operations were applied.
+ *
+ * @param operations - List of edit operations
+ * @returns Human-readable summary string
+ */
+export function narrateOperations(operations: EditOperation[]): string {
+  const narratives = operations.map((op) => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)